# Message definition for waypoints.

float32[] waypointOne
float32[] waypointTwo
float32[] waypointThree
float32[] waypointFour